dan_layer/consensus/src/hotstuff/on_receive_request_missing_transactions.rs

```diff
@@ -0,0 +1,55 @@
+//   Copyright 2023 The Tari Project
+//   SPDX-License-Identifier: BSD-3-Clause
+
+use log::*;
+use tari_dan_storage::{consensus_models::ExecutedTransaction, StateStore};
+use tokio::sync::mpsc;
+
+use crate::{
+    hotstuff::error::HotStuffError,
+    messages::{HotstuffMessage, RequestMissingTransactionsMessage, RequestedTransactionMessage},
+    traits::ConsensusSpec,
+};
+
+const LOG_TARGET: &str = "tari::dan::consensus::hotstuff::on_receive_request_missing_transactions";
+
+pub struct OnReceiveRequestMissingTransactions<TConsensusSpec: ConsensusSpec> {
+    store: TConsensusSpec::StateStore,
+    tx_leader: mpsc::Sender<(TConsensusSpec::Addr, HotstuffMessage)>,
```

Comment:
This might not only be the leader who processes this, but anyone who has it
```suggestion
    tx_request_missing_tx: mpsc::Sender<(TConsensusSpec::Addr, HotstuffMessage)>,
```

Reply:
yeah, the name was just an copy paste
renamed